app/build.gradle

```
@@ -52,7 +52,7 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation group: 'androidx.constraintlayout', name: 'constraintlayout', version: '1.1.3'
-    implementation 'io.sentry:sentry-android:2.3.1'
+    implementation 'io.sentry:sentry-android:3.1.0'
```

```suggestion
    implementation 'io.sentry:sentry-android:3.1.1'
```

https://docs.sentry.io/platforms/android/migration/#migrating-from-sentry-android-2x-to-sentry-android-3x

you can remove the line that enables the auto session tracking, this is enabled automatically now

---

https://github.com/sentry-demos/android/blob/master/app/src/main/AndroidManifest.xml#L48-L49

---

thanks, I removed that line :)